src/tools/yarprobotstatepublisher/src/main.cpp

```diff
@@ -35,6 +35,7 @@ int main(int argc, char *argv[])
     if (rf.check("help"))
     {
         cout<<"Options"<<endl;
+        cout<<"\t--name                   <name>: prefix of the yarprobotstatepublisher ports"<<endl;
```

Comment:
The code seem to be searching for a `namePrefix` option, not for a `name` one.

Reply:
I think the first two options should be updated as
- `name` ------> `namePrefix`
- `prefix` -------> `--tfPrefix`

This will reflect the arguments to be passed while launching `yarprobotstatepublisher`

e.g. `yarprobotstatepublisher --namePrefix Human --tfPrefix /Human/ --model Claudia66DoF.urdf --base-frame Pelvis --jointstates-topic "/Human/joint_states"`

what do you thinks @traversaro ?

Reply:
Ok for me. This commands where just used in the `robotology:feature/improvedRobotStatePublisher`, right?